ppl/cmd/zar/index/create.go

```diff
@@ -43,29 +44,32 @@ requires specifying the key and output file name. For example:
 
 type CreateCommand struct {
 	*root.Command
+	displayer  *displayer
```

Comment:
Make this `displayer displayer` so you don't have to initialize it?

Reply:
yup